src/test/java/com/acrolinx/sidebar/swing/AcrolinxSidebarSwingTest.java

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2017-present Acrolinx GmbH */
+
+package com.acrolinx.sidebar.swing;
+
+import java.awt.event.KeyEvent;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.acrolinx.sidebar.AcrolinxIntegration;
+
+class AcrolinxSidebarSwingTest
+{
+
+    private static void testPasteCommand(int key, int modifier, boolean consume)
```

Comment:
I would rename "key" to "keyCode"
And should not "modifier" be called "modifiers" since the method is called "getModifiers()"?

Reply:
`getModifiers()` returns an int